tap_aircall/client.py

```diff
@@ -112,8 +114,19 @@ def validate_response(self, response: requests.Response) -> None:
         try:
```

Comment:
can we actually remove this `validate_response` function entirely? I added it initially so that we could, in a sense, filter out the 429 errors from our alerts (ie. not raise an exception for them and instead just return a special exit code that the entrypoint.py file in the lloyd repo could detect and use to instead set the status of these tap runs that return with a 429 status code to be labeled as successes rather than failures).

Now that you have added the `backoff_wait_generator`, I don't think we should need the modified `validate_response` at all actually! Since the superclass will already handle raising the exceptions for all of the errors and we actually don't want to mark any of the other non-429 errors as "accepted intermittent errors" since whenever we do encounter them they are likely caused by problems within the tap and so we would want to alert that the tap is down correspondingly!